### ISSUE TITLE
Option to skip npm install in Azure/static-web-apps-deploy

### ISSUE THREAD

**Issue by sfabriece** [Azure/static-web-apps#484]
**Is your feature request related to a problem? Please describe.**
I have a github workflow that runs npm install explicitly. It would desirable to be able to skip this as part of the `Azure/static-web-apps-deploy` action

**Describe the solution you'd like**
A option passed to the action to disable `npm install`

**Describe alternatives you've considered**
Not using `Azure/static-web-apps-deploy` action

**Additional context**

**Comment by anthonychu** [comment]
Please see if the `skip_app_build` option works for you: https://docs.microsoft.com/en-us/azure/static-web-apps/github-actions-workflow#skip-app-build

**Comment by sfabriece** [comment]
I don't think that it will work, since the documentation says that npm install is always run. But I can give it a go.

**Comment by aaronpowell** [comment]
You can do this by overriding how Oryx generates the build script (https://github.com/microsoft/Oryx/blob/master/src/BuildScriptGenerator/Node/NodeBashBuildSnippet.sh.tpl), specifically you want to set the `CUSTOM_BUILD_COMMAND` environment variable.

If you set it to `npm run build`, Oryx will insert this block https://github.com/microsoft/Oryx/blob/master/src/BuildScriptGenerator/Node/NodeBashBuildSnippet.sh.tpl#L89-L93 which just does you're build and not `npm install --unsafe-perm` (which is the default).

I've only tested this in a really simplistic scenario, but it works _in theory_.

**Comment by dysticus** [comment]
I have a similar question. For my project, I have quite a custom build setup in comparison to what Oryx provides by default e.g. Install dependencies with Yarn, activate a license between the installation and the build, running unit tests before pruning out devdependencies,... 

For me it would make more sense to have the option to skip the entire build part (and handle it on my own) and take advantage of the build-in deployment part of the Github action. Is this something that is possible with this action at the moment?
